services/src/main/java/org/fao/geonet/services/rdf/GetPublicMetadataAsRdf.java

```diff
@@ -0,0 +1,33 @@
+package org.fao.geonet.services.rdf;
+
+import jeeves.interfaces.Service;
+import jeeves.server.ServiceConfig;
+import jeeves.server.context.ServiceContext;
+import org.fao.geonet.services.thesaurus.GetList;
+import org.fao.geonet.utils.BinaryFile;
+import org.jdom.Element;
+
+import java.io.File;
+import java.nio.file.Path;
+
+public class GetPublicMetadataAsRdf implements Service {
+    public void init(Path appPath, ServiceConfig params) throws Exception {}
+
+    //--------------------------------------------------------------------------
+    //---
+    //--- Service
+    //---
+    //--------------------------------------------------------------------------
+
+    public Element exec(Element params, ServiceContext context) throws Exception {
```

Comment:
Would be good to use Spring MVC for new services (and make them part of the API with doc) - we should discuss that for develop branch I think.